src-common/vcastServer.ts

```diff
@@ -0,0 +1,221 @@
+import fetch from "node-fetch";
+import { pythonErrorCodes } from "./vcastServerTypes";
+
+let HOST = "localhost"; // The server's hostname or IP address
+let PORT = 60461; // The port used by the server anything > 1023 is OK
+
+// Types used for interaction with the python interface and the enviro server
+//
+// Note: some of these match the --mode argument for the vTestInterface.py but
+// some of these (like ping) are for the enviro server only
+//
+// Note: this enum must stay in sync with the server file: vcastDataServerTypes.py: commandType.
+
```

Comment:
Do we think this enum will be changing frequently? Maybe we could have a `.json` file with this mapping that gets read by both `.ts` and the `.py` counterpart. If the enum doesn't really change often, it might not be worth moving the mapping to a json file

Reply:
I don't think we will be changing this very often.  I added a comment to both places with this commit: 1615e28

Reply:
All good then